src/models/scatter.js

```diff
@@ -396,63 +399,63 @@ nv.models.scatter = function() {
                         //.style('pointer-events', 'auto') // recativate events, disabled by css
                         .on('click', function(d,i) {
                             //nv.log('test', d, i);
-                            if (needsUpdate || !data[d.series]) return 0; //check if this is a dummy point
-                            var series = data[d.series],
+                            if (needsUpdate || !data[d[0].series]) return 0; //check if this is a dummy point
+                            var series = data[d[0].series],
```

Comment:
Why the [0] all over?  d should be the data object, it shouldn't be an array...

Reply:
If memory serves right, the error when using useVoronoi "false" was that the series property didn't exist; when I debugged, I found out that the real value was inside an array, hence the change.